### DISCUSSION TITLE
Release v1.9.7 / 5.64.7

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3025] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This fixes is a maintenance release it fixes a coupe issues and adds a few minor ui improvements.

Please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#197--5647---2023-06-09) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.7'>Release v1.9.7 / 5.64.7</a>.</em>

**Comment by bastik-1001** [comment]
The sentence is broken. "This fixes is a maintenance..." should be "This is a ..."

**Comment by BilleBarrett** [comment]
This the only thing that happens in Chrome "Ellen" is a client Running Windows 11 Pro.
This with Plus 1.9.4, I have tried 1.9.5, 1.9.6 and 1.9.7.

```
chrome.exe   7348                     Running (Chromium Based)   07:47:00   "C:\Program Files\Google\Chrome\Application\chrome.exe" --type=crashpad-handler "--user-data-dir=C:\Users\Ellen\AppData\Local\Google\Chrome\User Data" /prefetch:7 --monitor-self-annotation=ptype=crashpad-handler "--database=C:\Users\Ellen\AppData\Local\Google\Chrome\User Data\Crashpad" --url=https://clients2.google.com/cr/report --annotation=channel= --annotation=plat=Win64 --annotation=prod=Chrome --annotation=ver=114.0.5735.134 --initial-client-data=0x164,0x168,0x16c,0x118,0x170,0x7ffb44a4d9e0,0x7ffb44a4d9f0,0x7ffb44a4da00
```

I have gone over several system and compared ini files and cannot find a difference, this is frustrating, it all started with the Microsoft update KB5027231 & Chrome update Version 114.0.5735.134 (Official Build) (64-bit).
What's frustrating is there are some Windows 11 computers like mine that are not affected.
Microsoft Edge has no problem running.

**Reply by DavidXanatos** [reply]
@BilleBarrett can it be that your users are affected by this issue: https://github.com/sandboxie-plus/Sandboxie/issues/3040 caused by the update KB5027231 ?

**Reply by DavidXanatos** [reply]
As it seams the issue is not caused by sandboxie, but by the MSFT update kb5027231 https://www.bleepingcomputer.com/news/microsoft/windows-11-kb5027231-also-breaks-chrome-for-cisco-watchguard-edr-users/ and it affects a lot of other security software

**Reply by BilleBarrett** [reply]
It seems that all you have to do is open windows Setting > Apps > Default Apps > Goode Chrome
then click on Set default  and this resolves the issue
I also went done the list and wherever Edge was I open and switch to Chrome

**Reply by isaak654** [reply]
Alternatively, see https://github.com/sandboxie-plus/Sandboxie/issues/3040#issuecomment-1596126796
In order to be applied for all sandboxes, it should be placed under [GlobalSettings] in Sandboxie.ini.
